dbt State dev environment and user credentials (#9433)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## June 2026

> - **Enhancement**: You can now configure [dbt State](/docs/deploy/dbt-state-about) for the <Constant name="studio_ide" /> in the <Constant name="dbt_platform" /> UI. Enable it on your development environment as the default for your team, or set a personal override to turn it on or off for yourself without affecting other users. For more information, refer to [Enabling dbt State in the Studio IDE](/docs/deploy/dbt-state-setup#enabling-dbt-state-in-the-studio-ide).

Reviewer: sugg shortening
```suggestion
- **Enhancement**: You can now configure [dbt State](/docs/deploy/dbt-state-about) for the <Constant name="studio_ide" /> directly in the <Constant name="dbt_platform" /> UI &mdash; either as a team-wide default on your development environment, or as a personal override. For more information, refer to [Enabling dbt State in the Studio IDE](/docs/deploy/dbt-state-setup#enabling-dbt-state-in-the-studio-ide).
```
